apps/web/src/pages/auth/components/QuestionnaireForm.tsx

```diff
@@ -66,7 +66,7 @@ export function QuestionnaireForm() {
     const createDto: ICreateOrganizationDto = { ...rest, name: organizationName };
     const organization = await createOrganizationMutation(createDto);
     const organizationResponseToken = await api.post(`/v1/auth/organizations/${organization._id}/switch`, {});
-
+    localStorage.setItem('onboarding_modal', 'true');
```

Comment:
```suggestion
    localStorage.setItem('nv_onboarding_modal', 'true');
```
would be good to prefix our localstorage items, to make sure that we are not colliding with other tools :)